patches/tModLoader/Terraria/ModLoader/ModLoader.cs

```diff
@@ -176,11 +176,22 @@ private static void Load(CancellationToken token = default)
 			if (e.Data.Contains("contentType") && e.Data["contentType"] is Type contentType)
 				msg += "\n" + Language.GetTextValue("tModLoader.LoadErrorContentType", contentType.FullName);
 
-			Logging.tML.Error(msg, e);
-
-			foreach (var mod in responsibleMods)
+			foreach (var mod in responsibleMods) {
 				DisableMod(mod);
 
+				var dependents = from m in availableMods
+					where m.properties.modReferences.Any(reference => reference.mod.Equals(mod))
```

Comment:
`BuildProperties` has the `RefNames` method which will help here. We used explicit named args when passing constants, so you'll want to do `RefNames(includeWeak: false)` which makes it really clear that you're only considering strong references as required deps.